raiden/network/transport/matrix/transport.py

```diff
@@ -640,35 +639,22 @@ def _handle_invite(self, room_id: _RoomID, state: dict):
         if self._stop_event.ready():
             return
 
-        self.log.debug('Got invite', room_id=room_id)
```

Comment:
The intention of having this logging call here was to be able to catch potential cases of invites not getting handled properly. 
Most likely not an issue, but can be helpful when debugging from a log file.